suse_migration_services/units/reboot.py

```diff
@@ -21,20 +21,30 @@
 from suse_migration_services.command import Command
 from suse_migration_services.logger import log
 from suse_migration_services.defaults import Defaults
+from suse_migration_services.fstab import Fstab
 
 
 def main():
     """
     DistMigration reboot with new kernel
+
+    After the migration process is finished, the system reboots
+    unless the debug file /etc/sle-migration-service is set.
```

Comment:
Since the config file is now a real config file and no longer a flag, this wording needs to change.

`...unless debug: True is set in the ... configuration file.`